main.py

```diff
@@ -39,26 +39,40 @@ def hello():
 
 
 @app.route('/circles/api/v1.0/users/<int:person_id>/', defaults={'resource': None})
-@app.route('/circles/api/v1.0/users/<int:person_id>/<resource>', methods=['GET'])
+@app.route('/circles/api/v1.0/users/<int:person_id>/<resource>', methods=['GET', 'POST'])
```

Comment:
I'm not sure this is the right endpoint for POST. Think it should be the top one `/<int:person_id>/`with the defaults. So they make a POST request to the top endpoint and include the details of the Person in the JSON.

Reply:
Actually, should be a new route: `@app.route('/circles/api/v1.0/users', methods=['POST'])`